<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountingBillPriceDataGUI</class>
 <widget class="QWidget" name="AccountingBillPriceDataGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dati computo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="generalDataSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="priceGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Opere a misura - Prezzi</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="priceListComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="priceListComboBoxLabel">
          <property name="text">
           <string>Prezzario</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="currentPriceDataSetSpinBoxLabel">
          <property name="text">
           <string>N. Preventivo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="currentPriceDataSetSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="discountLabel">
          <property name="text">
           <string>Ribasso</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="discountLineEdit">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="pricesGroupBox">
          <property name="title">
           <string>Campi</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QGroupBox" name="totalPriceFieldGroupBox">
              <property name="title">
               <string>Importo lordo</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="QTableView" name="totalPriceFieldTableView"/>
               </item>
              </layout>
             </widget>
             <widget class="QGroupBox" name="noDiscountPriceFieldGroupBox">
              <property name="title">
               <string>Importo non ribassabile</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QTableView" name="noDiscountPriceFieldTableView"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
